### PR TITLE
Update application credentials instructions for google calendar OAuth

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Update incorrect google calender oauth instructions based on discussion #70241

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
@allenporter There is a merge conflict, could you take a look? 👍

**Comment by dannytsang** [comment]
@allenporter looks good. Very minor comments and it covers https://github.com/home-assistant/home-assistant.io/pull/23270 as well. Thank you for updating it.

File: source/_integrations/google.markdown [hunk 1]
```diff
@@ -81,6 +83,8 @@ to all the Google Calendars available to your linked account.
 
 If the setup process fails and you see an error message such as *Authentication code expired, please try again* you may want to try the flow again. You may also check the logs for additional error messages that may indicate a misconfiguration such as an invalid client id or secret.
 
+If you have an error with your credentails you can delete them in the [Application Credentials](/integrations/application_credentials/) user interface.
```
Comment (frenck) [inline source/_integrations/google.markdown]:
```suggestion
If you have an error with your credentials you can delete them in the [Application Credentials](/integrations/application_credentials/) user interface.
```

File: source/_integrations/google.markdown [hunk 2]
```diff
@@ -28,24 +28,26 @@ This section explains how to generate a Client ID and Client Secret on
...
+1. Click on the field on the right of the screen, **OAuth Consent Screen**.
+1. Select **External** and **Create**.
+1. Set the *App Name* (the name of the application asking for consent) to anything you want e.g.  *Home Assistant*.
+1. You then need to select a *Support email*. To do this, simply click the drop down box and select your email address.
+1. You finally need to complete the section: *Developer contact information*. To do this, simply enter your email address (same as above is fine).
+1.  Scroll to the bottom and click **Save and Continue**. Don't have to fill out anything else or it may enable additional review.
+1.  You will then be automatically taken to the *Scopes* page. You do not need to add any scopes here so click **Save and Continue** to move to the *Test users* page. You will need to add the Gmail address you will be using with this integration as a test user before you will be allowed to use the API. Once you have added your email address under *Test users* click **Save and Continue** which will take to the *Summary* page. (you do not need to do anything here).
```
Comment (frenck) [inline source/_integrations/google.markdown]:
```suggestion
1.  You will then be automatically taken to the *Scopes* page. You do not need to add any scopes here so click **Save and Continue** to move to the *Test users* page. You will need to add the Gmail address you will be using with this integration as a test user before you will be allowed to use the API. Once you have added your email address under *Test users* click **Save and Continue** which will take you to the *Summary* page. (you do not need to do anything here).
```

**Review by frenck — APPROVED** [review]
Thanks, @allenporter 👍

**Comment by rhino53150** [comment]
"Generate Client ID and Client Secret" instructions are not correct. I just stepped through the process twice and step 11 states "You will then be automatically taken to the Scopes page. You do not need to add any scopes here so click Save and Continue to move to the Test users page." There is an "Optional Info" page after the "Scopes" page and then "Summary"


2022-07-09 13_49_59-Window

**Comment by allenporter** [comment]
Can you send an edit with what makes sense to you?

**Comment by rhino53150** [comment]
> Can you send an edit with what makes sense to you?

11. You will then be automatically taken to the Scopes page. You do not need to add any scopes here so click Save and Continue to move to the Optional info page. You do not need to add anything to the Optional info page so click Save and Continue which will take you to the Summary page. Click Back to Dashboard.

**Comment by allenporter** [comment]
> > Can you send an edit with what makes sense to you?
> 
> 11. You will then be automatically taken to the Scopes page. You do not need to add any scopes here so click Save and Continue to move to the Optional info page. You do not need to add anything to the Optional info page so click Save and Continue which will take you to the Summary page. Click Back to Dashboard.

I meant send an actual edit PR e.g. https://community.home-assistant.io/t/editing-the-documentation-and-creating-a-pull-request-on-github/9573

Thank you for your contributions!

**Pull request merged**